### PR TITLE
feat!: Boxing Day

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

* Box spent the summer in France with cousins returned as an international polygon of mystery, and is now the coolest freshman in the history of CC High.
* FlexBox and GridBox were there too I guess - but they are just younger siblings who follow Box around - v uncool

### PR Checklist

- [x] Related to designs: [Box Is Everywhere This Season](https://codecademy.com)
- [x] Related to JIRA ticket: [BOX-2](#)
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

#### Warning 😨 
This is 100% **NOT** mergable - but can serve as a reference PR there are incompatibilities between variance and animus theme shapes the break responsive styles. 

I can create a migration guide if there interest in it.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.105-alpha.618a61.0<br/>@codecademy/gamut-labs@31.8.1-alpha.618a61.0<br/>@codecademy/gamut@46.1.2-alpha.618a61.0<br/>@codecademy/monaco@0.6.34-alpha.618a61.0<br/>@codecademy/styleguide@56.4.4-alpha.618a61.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6259a86718137405749a8e11--gamut-preview.netlify.app>https://6259a86718137405749a8e11--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6259a86718137405749a8e11>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.106-alpha.7b658b.0<br/>@codecademy/gamut-labs@31.8.2-alpha.7b658b.0<br/>@codecademy/gamut@46.1.3-alpha.7b658b.0<br/>@codecademy/monaco@0.6.35-alpha.7b658b.0<br/>@codecademy/styleguide@56.4.5-alpha.7b658b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6259aaa44a8d110689c1dffa--gamut-preview.netlify.app>https://6259aaa44a8d110689c1dffa--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6259aaa44a8d110689c1dffa>Deploy Logs</a></p>

File: packages/gamut/src/Box/Box.tsx [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+    borders: true,
+    layout: true,
+    color: true,
+    shadows: true,
+    positioning: true,
+    background: true,
+    typography: true,
+    flex: true,
+    grid: true,
+  })
```
Comment (dreamwasp) [inline packages/gamut/src/Box/Box.tsx:29]: we love to see it

File: packages/gamut/src/Card/index.tsx [hunk 2]
```diff
@@ -5,8 +5,8 @@ import React, { ComponentProps } from 'react';
 
 import { Box } from '../Box';
 
-const DynamicCardWrapper = styled(Box)<CardWrapperProps>(
-  variant({
+const DynamicCardWrapper = Box.extend()
+  .variant({
```
Comment (dreamwasp) [inline packages/gamut/src/Card/index.tsx:9]: https://media.giphy.com/media/sBl8Fowq0ErFC/giphy.gif

**Review by dreamwasp — COMMENTED** [review]
tre cool

**Comment by jakemhiller** [comment]
❤️ u @codecaaron

**Pull request closed without merging**